junit-platform-gradle-plugin/src/test/groovy/org/junit/platform/gradle/plugin/JUnitPlatformPluginSpec.groovy

```diff
@@ -63,6 +64,8 @@ class JUnitPlatformPluginSpec extends Specification {
 			}
 
 			reportsDir new File("any")
+
+			details Details.HIDDEN
```

Comment:
No. It is case insensitive. 

See [EnumConverter](https://github.com/pholser/jopt-simple/blob/master/src/main/java/joptsimple/util/EnumConverter.java#L58) from jopt-simple.